hr_employee_partner/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name":     "Employee Partner",
+    "summary":  "Associate a Partner to Employee",
+    "version":  "12.0.1.0.0",
+
+    "category": "Human Resources",
+
+    "author":   "Stefano Consolaro Associazione PNLUG - Gruppo Odoo, "
+                "Odoo Community Association (OCA)",
+    "website":  "https://github.com/OCA/hr",
+    "license":  "AGPL-3",
+
+    "depends": [
+        'hr',
+        ],
+    "data": [
+        'views/hr_employee.xml',
+        ],
+    'installable': True,
+    'auto_install': False,
```

Comment:
Not required auto_install and installable

Reply:
Many things of Odoo are still on the dark side for me :-). Thanks